server/__init__.py

```diff
@@ -103,10 +104,20 @@ def report_dirties():
             player_service.clear_dirty()
 
             if len(dirty_queues) > 0:
-                ctx.broadcast_raw(encode_queues(dirty_queues), lambda lobby_conn: lobby_conn.authenticated)
+                asyncio.ensure_future(
+                    ctx.broadcast_raw(
+                        encode_queues(dirty_queues),
+                        lambda lobby_conn: lobby_conn.authenticated
+                    )
+                )
```

Comment:
This will cause `report_dirties` to not wait for these broadcasts. Worst case scenario, we'll start doing these in parallel. We need to change `report_dirties` to a coroutine as well. Also an opportunity to make its infinite loop sane.

Reply:
I just took a look at `aiocron` and this is pretty much exactly how it is implemented. It has a synchronous function which fires off a coroutine without waiting for it to complete, and then schedules its self to be called again via `loop.call_at`. I tried working on refactoring it to use more or less of a sleep loop, but then you need to have a way of signaling the loop to shut down when the server shuts down, which is really annoying to do. With the call_at method, you don't need a shutdown signal, because if the server shuts down then the coroutine just won't be called again.